data/data/openstack/topology/sg-lb.tf

```diff
@@ -23,22 +23,12 @@ resource "openstack_networking_secgroup_rule_v2" "api_https" {
   security_group_id = "${openstack_networking_secgroup_v2.api.id}"
 }
 
-resource "openstack_networking_secgroup_rule_v2" "api_ingress_dns_udp" {
-  direction         = "ingress"
-  ethertype         = "IPv4"
-  protocol          = "udp"
-  port_range_min    = 53
-  port_range_max    = 53
-  remote_ip_prefix  = "0.0.0.0/0"
-  security_group_id = "${openstack_networking_secgroup_v2.api.id}"
-}
-
-resource "openstack_networking_secgroup_rule_v2" "api_ingress_dns_tcp" {
+resource "openstack_networking_secgroup_rule_v2" "api_ingress_ssh_tcp" {
```

Comment:
After a rebase on latest master, this rule gets duplicated, which causes terraform to fail.  It'll need to be removed.